plugin/trino-influxdb/src/test/java/io/trino/plugin/influxdb/client/TestInfluxDataType.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb.client;
+
+import io.trino.spi.type.BigintType;
+import io.trino.spi.type.BooleanType;
+import io.trino.spi.type.DateType;
+import io.trino.spi.type.DoubleType;
+import io.trino.spi.type.VarcharType;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.influxdb.client.InfluxDataType.BOOLEAN;
+import static io.trino.plugin.influxdb.client.InfluxDataType.DATE;
+import static io.trino.plugin.influxdb.client.InfluxDataType.FLOAT;
+import static io.trino.plugin.influxdb.client.InfluxDataType.INTEGER;
+import static io.trino.plugin.influxdb.client.InfluxDataType.STRING;
+import static io.trino.plugin.influxdb.client.InfluxDataType.TIME;
+import static io.trino.plugin.influxdb.client.InfluxDataType.TYPES_MAPPING;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNull;
+
+public class TestInfluxDataType
```

Comment:
Can we have a detailed type mapping test like we have in Druid or Pinot or other JDBC based connector.